plugin.rb

```diff
@@ -497,4 +497,29 @@ def self.enqueue_assign_hooks(event, payload)
     end
   end
 
+  require_dependency 'search'
+
+  #TODO Remove when plugin is 1.0
+  if Search.respond_to? :advanced_filter
+    Search.advanced_filter(/in:assigned/) do |posts|
```

Comment:
I think you should be able to replace all this with the [plugin API](https://github.com/discourse/discourse/blob/master/lib/plugin/instance.rb#L205-L205)
```suggestion
register_search_advanced_filter(/in:assigned/) do |posts|
```